cidc_api/models.py

```diff
@@ -624,6 +638,23 @@ def find_by_id_and_email(cls, id, email, session):
             return None
         return upload
 
+    @with_default_session
+    def ingestion_success(self, session, commit=False):
+        """Set own status to reflect successful merge and trigger email notifying CIDC admins."""
+        # Do status update if the transition is valid
+        if not AssayUploadStatus.is_valid_transition(
```

Comment:
with `self` being `AssayUploads` I think it might be more appropriate to call this as an instance method.

Reply:
`is_valid_transition` doesn't exist as an instance method on `AssayUploads`, only as a static method on the `AssayUploadStatus` enum.

Reply:
```
In [4]: class What():
   ...:     ever = "nooo"
   ...:     @classmethod
   ...:     def yeah(cls):
   ...:         return cls.ever
   ...:

In [5]: What().yeah()
Out[5]: 'nooo'
```